docs/content/latest/explore/colocated-tables.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Colocated tables
+linkTitle: Colocated tables
+description: Colocated tables
+menu:
+  latest:
+    identifier: colocated-tables
+    parent: explore
+    weight: 245
+isTocNested: true
+showAsideToc: true
+---
+
+Colocating tables can dramatically increase the number of relations (tables, indexes, etc) that can
```

Comment:
Since this is a new topic to explore, give a short definition of what the feature is and why it's useful. Then discuss the issues using it.